### ISSUE TITLE
Netatmo CO2 UnhandledPromiseRejectionWarning

### ISSUE THREAD

**Issue by Veldkornet** [swttt/com.swttt.homekit#149]
HomeyKit v3.0.4
Homey v2.0.5

It seems as though every time the CO2 level is measured, it's throwing an exception (although it does fetch the value):
```
State Change - Binnen - measure_humidity - 49
State Change - Binnen - measure_pressure - 1036.2
State Change - Binnen - measure_co2 - 599
(node:25802) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'getCharacteristic' of undefined
    at onStateChange (/lib/devices/sensor.js:235:9)
    at e.listener [as _listener] (/lib/devices/sensor.js:163:9)
    at e.value (/node_modules/athom-api/dist/index.js:1:1062974)
    at o (/node_modules/athom-api/dist/index.js:1:1088690)
    at t.value (/node_modules/athom-api/dist/index.js:1:1092528)
    at t.<anonymous> (/node_modules/athom-api/dist/index.js:1:1040999)
    at a (/node_modules/athom-api/dist/index.js:1:324246)
    at Generator._invoke (/node_modules/athom-api/dist/index.js:1:325288)
    at Generator.e.(anonymous function) [as next] (/node_modules/athom-api/dist/index.js:1:324425)
    at n (/node_modules/athom-api/dist/index.js:1:8623)
(node:25802) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
```

Looking in my HomeKit on the phone, my Netatmo only shows temperature, humidity.
Pressure and CO2 seem to be missing (although pressure does not give the error).

**Comment by Veldkornet** [comment]
It does find 4 out of the 5 sensors when HomeyKit starts (sound being the 5th one):

```
2019-03-26 12:43:38 [log] [HomekitApp] Found Sensor: Binnen
created listener for - measure_temperature
created listener for - measure_co2
created listener for - measure_humidity
created listener for - measure_pressure
```

**Comment by robertklep** [comment]
Should be fixed in upcoming v3.0.10

**Comment by Veldkornet** [comment]
Yup, looks like it's fixed in 3.0.10.
I also see my CO2 sensor now in HomeKit which it wasn't doing before.

Any reason why the pressure is not showing up? I see it's being measured.

Also, may be out of your control (Ie, a HomeKit issue), but is there any way to get the CO2 value displayed in the tile itself? Currently, you need to go to the sensor settings to see it.

**Comment by robertklep** [comment]
The iOS Home app doesn't support air pressure, it's a custom characteristic that's only supported by some alternative Homekit apps (like [Eve](https://itunes.apple.com/us/app/eve-for-homekit/id917695792?mt=8) or [Home 3](https://itunes.apple.com/us/app/home-3/id995994352?mt=8)).
